requirements.txt

```diff
@@ -71,4 +71,6 @@ django-rest-durin==0.1.0
 dnstwist==20201022
 ppdeep>=20200505
 google>=3.0.0
-eventlet==0.29.1
\ No newline at end of file
+eventlet==0.29.1
+django-nose==1.4.7
+coverage==5.3.1
```

Comment:
are these new dependencies required for production? I guess not. Those should be removed from here

Reply:
well, the `django-nose` package needs to be under `INSTALLED_APPS` in `settings,py`. @m0mosenpai but we don't need the `coverage` package in the production requirements file so it can be removed.